### PR TITLE
Update PayPal note

### PR DESCRIPTION
**Pull request by person1234-ux**
I went to enable a u2f on paypal and they wouldn't let me enable it and then forced me to login using only sms. so I added some not so nice notes

### CODE REVIEW

File: entries/p/paypal.com.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "totp",
       "u2f"
     ],
-    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information.",
+    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information. (2fa isn't usefull for paypal as they default to single factor sms)",
```
Comment (Carlgo11) [inline entries/p/paypal.com.json:13]: Looks like TOTP is also supported as a backup method.

```suggestion
    "notes": "Some methods of 2FA are only available in a few countries. Contact PayPal for more information. U2F/WebAuthn requires a backup authentication method",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/p/paypal.com.json (hunk 1 above)
Comment (person1234-ux) [reply entries/p/paypal.com.json:13]: @Carlgo11 I realized  I need to give more context paypal current state doesn't actually meet the requirements of 2fa as they allow you to login via sms as the only factor on any device even if it wasn't previously used. This was my experience yesterday that lead me to closing my account

Comment (Carlgo11) [reply entries/p/paypal.com.json:13]: @person1234-ux I tried to reproduce what you described but the result differs from yours.

If I enter my correct password I get the following options:
Screenshot 2022-10-31 at 00-00-09 PayPal

If I click on _forgot password_ or _Having trouble logging in?_ I indeed get the option of SMS authentication.
If we're to assume that factor gets compromised (due to SIM-swapping) I still get the following requirement after successfully entering the SMS code:
Screenshot 2022-10-31 at 00-07-17 PayPal

So by that process the authentication factors are the same but the methods are weaker:
|Circumstance|Knowledge method|Possession method|
|-|-|-|
|Normal Authentication|Password|U2F Key|
|Password Reset|Security Questions|SMS code|

Of course SMS + Questions are far from ideal but technically 2FA is still enforced.

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**